src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const keyAndValueArray
+  = sourceString.split(';')
```

Comment:
```suggestion
  const keyAndValueArray = sourceString
    .split(';')
```
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/45087e3f-9099-4428-820e-c9d46874aaac)